functions/imagessr.js

```diff
@@ -2,34 +2,76 @@
 // eslint-disable-next-line import/no-unresolved
 const { onRequest } = require('firebase-functions/v2/https');
 const express = require('express');
-const axios = require('axios');
-const nodeHtmlToImage = require('node-html-to-image');
+const chromium = require('chrome-aws-lambda');
+const puppeteer = require('puppeteer-core');
 const sharp = require('sharp');
 
+const RESOURCES_TIMEOUT = 10000; // 10s
+
+let globalBrowser = null;
+
+async function getBrowser() {
+  const browser = await puppeteer.launch({
+    args: chromium.args,
+    defaultViewport: chromium.defaultViewport,
+    executablePath: await chromium.executablePath,
+    headless: chromium.headless,
+  });
+  return browser;
+}
+
+async function getNewPage(browser, { width = 1024, height = 768 } = {}) {
+  const page = await browser.newPage();
+  page.setDefaultNavigationTimeout(RESOURCES_TIMEOUT);
+  page.setDefaultTimeout(RESOURCES_TIMEOUT);
+  await page.setUserAgent('LikeCoin Button Image SSR');
+  await page.setViewport({ width, height });
+  return page;
+}
+
+async function makeScreenShot(page, url, {
```

Comment:
screenshot should one word?
```suggestion
async function makeScreenshot(page, url, {
```